medcat/utils/meta_cat/data_utils.py

```diff
@@ -117,10 +173,12 @@ def encode_category_values(data: Dict, existing_category_value2id: Optional[Dict
             Output of `prepare_from_json`.
         existing_category_value2id(Optional[Dict]):
             Map from category_value to id (old/existing).
+        category_undersample:
+            Name of class that should be used to undersample the data (for 2 phase learning)
 
     Returns:
         dict:
-            New data with integeres inplace of strings for categry values.
+            New data with integers inplace of strings for categry values.
```

Comment:
Looks like this now returns a tuple of 3 things. Would be great to document that.

Reply:
I've added it in the docstrings